### PR TITLE
[Feature] Rebrand sample app with Contoso branding

### PR DESCRIPTION
**Pull request by iseabock**
This PR adds a Contoso logo and rebrands the app with Contoso branding.

Screenshot 2024-01-09 113013

Note: I've commented on the changes regarding branding changes. Everything else is either clean up for formatting.

### CODE REVIEW

File: frontend/src/pages/layout/Layout.tsx [hunk 1]
```diff
@@ -1,38 +1,13 @@
 import { Outlet, Link } from "react-router-dom";
 import styles from "./Layout.module.css";
-import Azure from "../../assets/Azure.svg";
```
Comment (iseabock) [inline frontend/src/pages/layout/Layout.tsx:3]: Relevant rebranding

File: frontend/src/pages/layout/Layout.tsx [hunk 2]
```diff
@@ -64,64 +39,62 @@ const Layout = () => {
...
     return (
         <div className={styles.layout}>
             <header className={styles.header} role={"banner"}>
-                <Stack horizontal verticalAlign="center" horizontalAlign="space-between"
-                // className={styles.headerContainer}
-                >
+                <Stack horizontal verticalAlign="center" horizontalAlign="space-between">
                     <Stack horizontal verticalAlign="center">
                         <img
-                            src={Azure}
```
Comment (iseabock) [inline frontend/src/pages/layout/Layout.tsx:77]: Relevant rebranding

File: frontend/src/pages/layout/Layout.tsx [hunk 3]
```diff
@@ -64,64 +39,62 @@ const Layout = () => {
...
+                            src={Contoso}
                             className={styles.headerIcon}
                             aria-hidden="true"
                         />
                         <Link to="/" className={styles.headerTitleContainer}>
-                            <h1 className={styles.headerTitle}>Azure AI</h1>
```
Comment (iseabock) [inline frontend/src/pages/layout/Layout.tsx:82]: Relevant rebranding

File: frontend/src/pages/chat/Chat.tsx [hunk 4]
```diff
@@ -9,7 +9,7 @@ import uuid from 'react-uuid';
 import { isEmpty } from "lodash-es";
 
 import styles from "./Chat.module.css";
-import Azure from "../../assets/Azure.svg";
```
Comment (iseabock) [inline frontend/src/pages/chat/Chat.tsx:12]: Relevant rebranding

File: frontend/src/pages/chat/Chat.tsx [hunk 5]
```diff
@@ -569,32 +569,32 @@ const Chat = () => {
...
+                    <h2 className={styles.chatEmptyStateSubtitle} style={{ fontSize: "20px" }}><strong>Authentication configuration takes a few minutes to apply. </strong></h2>
+                    <h2 className={styles.chatEmptyStateSubtitle} style={{ fontSize: "20px" }}><strong>If you deployed in the last 10 minutes, please wait and reload the page after 10 minutes.</strong></h2>
                 </Stack>
             ) : (
                 <Stack horizontal className={styles.chatRoot}>
                     <div className={styles.chatContainer}>
                         {!messages || messages.length < 1 ? (
                             <Stack className={styles.chatEmptyState}>
                                 <img
-                                    src={Azure}
```
Comment (iseabock) [inline frontend/src/pages/chat/Chat.tsx:589]: Relevant rebranding

File: static/index.html [hunk 6]
```diff
@@ -5,7 +5,7 @@
     <link rel="icon" type="image/x-icon" href="/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Azure AI</title>
```
Comment (sarah-widder) [inline static/index.html:7]: Probably need to change this to Contoso

**Review by sarah-widder — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**